src/features/locators/components/LocatorEditDialog.tsx

```diff
@@ -84,6 +90,11 @@ export const LocatorEditDialog: React.FC<Props> = ({
 
   const nameValidationRules: Rule[] = createNameValidationRules(_isNameUnique);
 
+  const resetAndCloseDialog = useCallback(() => {
```

Comment:
the same: what if we will remove `reset` from this implementation and handle closing some other way? It would be better not to change the entire code

Reply:
why do you need callback here? what does it really affects?